presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcResultSet.java

```diff
@@ -193,6 +225,45 @@ public void testObjectTypes()
             assertEquals(rs.getTimestamp(column), Timestamp.valueOf(LocalDateTime.of(2018, 2, 13, 13, 14, 15, 555_555_556)));
         });
 
+        // distant past, but apparently not an uncommon value in practice
+        checkRepresentation("TIMESTAMP '0001-01-01 00:00:00'", Types.TIMESTAMP, (rs, column) -> {
+            assertEquals(rs.getObject(column), Timestamp.valueOf(LocalDateTime.of(1, 1, 1, 0, 0, 0)));
```

Comment:
I realize the others do this, but it'd be better to factor this out
```java
Timestamp expected = Timestamp.valueOf(LocalDateTime.of(1, 1, 1, 0, 0, 0);
```
That way it's clear to the reader that these are the same value.

Reply:
Others do it to avoid sharing a mutable objected as the expected state.